src/main/java/seedu/address/logic/commands/AddClaimCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLAIM_AMOUNT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLAIM_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INSURANCE_ID;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a claim to an existing person with existing Insurance Plan in the address book.
+ */
+public class AddClaimCommand extends Command {
+    public static final String COMMAND_WORD = "addClaim";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds an open claim to the insurance plan of the client identified "
+            + "by their client id. \n"
+            + "Parameters: INDEX (must be a positive integer), "
+            + " INSURANCE_PLAN_ID (must be a valid ID), "
+            + " Claim_ID (must be a valid ID), "
+            + " Claim amount (must be a valid integer, convert from dollars and cents to just cents) \n"
```

Comment:
Should claim amount be in dollars and cents when the user inputs it, while we save claim amount as cents in the system? The display of claim amount on the GUI to also be in dollars and cents for easier understanding. This way the app can be more user friendly. What do you think?

Reply:
I agree, addressing this in PR #96 as well, in the parser, we should allow for input in the form of 30.15 (as in $30.15) and convert it to integer 3015. It's more intuitive for the user to enter the amount that way.

Reply:
fixed in latest PR, will be linked later.